src/mjolnir/graphtilebuilder.cc

```diff
@@ -1021,11 +1021,8 @@ void GraphTileBuilder::AddBins(const std::string& tile_dir,
 
 // Add a predicted speed profile for a directed edge.
 void GraphTileBuilder::AddPredictedSpeed(const uint32_t idx,
-                                         const std::vector<int16_t>& profile,
+                                         const std::array<int16_t, kCoefficientCount>& coefficients,
```

Comment:
it was stupid to take a vector when we wanted the size of the thing to be exactly a certain length. std::array does this. so lets use that instead